Source/gs/GSH_OpenGL/GSH_OpenGL.cpp

```diff
@@ -844,6 +844,11 @@ void CGSH_OpenGL::SetupBlendingFunction(uint64 alphaReg)
 		//2101 -> Cd * (1 - As)
 		glBlendFuncSeparate(GL_ZERO, BLEND_ONE_MINUS_SRC_ALPHA, GL_ONE, GL_ZERO);
 	}
+	else if((alpha.nA == ALPHABLEND_ABD_ZERO) && (alpha.nB == ALPHABLEND_ABD_CS) && (alpha.nC == ALPHABLEND_C_AS) && (alpha.nD == ALPHABLEND_ABD_CS))
```

Comment:
Can you move this block between the implementation for 1222 and 2001? I'm trying to keep those sorted.